pkg/config/config.go

```diff
@@ -8,8 +8,8 @@ import (
 	"flag"
 	"io/ioutil"
 
+	"go.uber.org/zap"
 	ini "gopkg.in/ini.v1"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
I understand circle dependency issue you worry about, we don't need to import "github.com/vmware-tanzu/nsx-operator/pkg/logger", I mean do we still use "sigs.k8s.io/controller-runtime/pkg/log"lib to create another logger, instead of using a zap logger, is it possible?